### DISCUSSION TITLE
Pressed & Hover States in destructive actions

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#860] [Proposals]
A raiz de que tenemos que revisar los actuales estados :active y :hover y que tenemos nuevos componentes con acciones destructivas como las action sheets y en breve, el menu dropdown. De cara a la creación de un futuro backgroundPressed (o como le llamemos) lanzo aquí la posibilidad de valorar para acciones destructivas un backgroundPressedDestructive que refuerce la acción.
Os pongo ejemplo visual para que quede más clara la idea.
<img width="1455" alt="6" src="https://user-images.githubusercontent.com/88187691/197202535-b7b6bea3-8117-4290-915e-81bed2be8c88.png">

Relacionado con la #677

**Comment by aweell** [comment]
+1 A esto! Consideraría que tanto el `hover` como el `pressed` vengan del pepper, usando por ejemplo el 10 y el 20? El 10 se esta usando actualmente en las light como error low, pero en dark se usa un blanco con alpha...habría que ver el dark como funciona

**Reply by AnaMontes11** [reply]
En ese ejercicio hice la prueba con el pepper10 para el background del hover/pressed y para el texto el pepper. Habría que probarlo con detalle.

**Comment by aweell** [comment]
I close this discussion since there's already a issue covering this topic #1203
